circonus/circonus_test.go

```diff
@@ -87,7 +87,7 @@ func TestSetGauge(t *testing.T) {
 		cs.Flush()
 	}()
 
-	expect := "{\"foo`bar\":{\"_type\":\"n\",\"_value\":1}}"
+	expect := "{\"foo`bar\":{\"_type\":\"n\",\"_value\":\"1\"}}"
```

Comment:
that's weird, maybe it's a vendor discrepancy? I didn't change the code getting called by this test so I thought it was broken